tests/main.ts

```diff
@@ -1,6 +1,8 @@
 import './unit/imports/lib/calendarTimeFormat';
 import './unit/imports/lib/relativeTimeFormat';
-import './unit/imports/server/MigrationRegistry';
+if (Meteor.isServer) {
+  import './unit/imports/server/MigrationRegistry';
+}
```

Comment:
Do we need both this and the `if` in the test file?

Reply:
No.  I already removed the condition from the test file in 793b7236e7e11e283563af6865692eaf9e68f7fd though.